dali/aux/optical_flow/optical_flow_adapter.h

```diff
@@ -50,6 +50,8 @@ struct OpticalFlowParams {
 
 using dali::kernels::TensorView;
 
+struct unsupported_exception : std::exception {};
```

Comment:
This is a bit generic.

Reply:
It's in optical_flow namespace, what's I guess is descriptive enough